modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -185,6 +185,127 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-171799 -  Verify it's possible to create a notification action with the trigger On Payment Status Update"
+	@priority = "5"
+	test CanCreateActionNotificationWithOnPaymentStatusUpdate {
+		property portal.acceptance = "true";
+
+		task ("Given a Notification Template is created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Notification Template");
+
+			Button.clickSave();
+		}
+
+		task ("When a notification action with the trigger On Order Status Update is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Commerce Order");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Custom Action",
+				active = "true",
+				inputExpression = "orderStatus == 1",
+				notificationName = "Notification Template",
+				thenAction = "Notification",
+				whenAction = "On Payment Status Update");
+		}
+
+		task ("And when an account, minium site and an order is created") {
```

Comment:
These can all be "And Given" instead of When. This is all still setup for the test